samples/sample_multi_transcode/src/sample_multi_transcode.cpp

```diff
@@ -358,90 +359,112 @@ void Launcher::DoTranscoding()
 {
     mfxStatus sts = MFX_ERR_NONE;
 
-    // get parallel sessions parameters
-    mfxU32 totalSessions = (mfxU32)m_pSessionArray.size();
-    MSDKThread* pthread = 0;
-
-    for (mfxU32 i = 0; i < totalSessions; i++)
+    // Helper structure to manage transcoding threads/sessions
+    struct Handles
     {
-        pthread = new MSDKThread(sts, TranscodeRoutine, (void *)m_pSessionArray[i]);
-        m_HDLArray.push_back(pthread);
-    }
+        std::future<void>       thrh;
```

Comment:
[thrh] - what does this name means?

Reply:
'thread handle' as @dvrogozh proposed. I can rename 'thrh' -> 'thrHandle'. Will be it OK?

Reply:
But this is [future] not thread handle.

Reply:
The future handles the return from the thread. It is common naming (it is used in samples for std::async). But I can rename it